app/controllers/provider_interface/decisions_controller.rb

```diff
@@ -33,76 +33,6 @@ def create
       end
     end
 
-    def respond
```

Comment:
Likewise for these actions, can we remove the views?

Reply:
Removed!